### ISSUE TITLE
SOGNet training phase and training details

### ISSUE THREAD

**Issue by Seolen** [LaoYang1994/SOGNet#2]
SOGNet inspires me a lot in modeling overlap relations for segmentation. There are two questions that I am confused.

1. Training phase: 
  In Figure 2 of the paper, instance ground truths are provided as input for training SOGNet. But the overlap resolving module requires mask logits from the instance head, how to match predicted mask logits and ground truths (their number, category and masks may be different)? Or how to obtain groundtruth mask logits?

2. Instances that have no overlap
  The paper writes "in implementations, we filter out the instances that have no overlap with any other instance...", hence in training and inference phases, the actual size of relation embedding E is the shape of (MXM, d), where M is the number of instances that have overlap with anyone. Right?

**Comment by LaoYang1994** [comment]
@Seolen  
hi，Thanks for your attention. 
+ For the first question
You can get the GT instance mask logits by feed the GT bbox into the mask branch to get the GT instance mask logits.
+ For the second question
M is the number of instances that overlap with at least one of the rest of instances.

The code will be released soon in about one or two weeks.
